web/packages/teleterm/src/ui/Search/pickers/ActionPicker.tsx

```diff
@@ -120,7 +118,13 @@ export function ActionPicker(props: { input: ReactElement }) {
           </Text>
         )}
       </EmptyListCopy>
-    ) : null;
+    ) : (
+      <EmptyListCopy>
+        <Text typography="subtitle1" color="text.primary">
+          Type something to search.
```

Comment:
idk how I feel about this, the unfortunate downside is that this will be shown only after the input debounces and not immediately.

Reply:
K, let's keep it then, maybe we'll change it in the next version.

Reply:
Actually, maybe we can approach this in a different way.

`NoResultsComponent` is used only by `ActionPicker` anyway. Instead of changing the API of `ResultList` to suit a very specific scenario of one specific component, we should let the users of `ResultList` to put an arbitrary stuff at the top of the result list.

Here you need it to tell the user which clusters were omitted from search results. Passing an arbitrary stuff to `ResultList` will also solve the problem with "Type something" appearing only after debounce – that arbitrary component will no longer have to be bound to the statuses of attempts.

In the PR with error handling, I'll need something like that anyway to show the errors. All those cases I talked about in this comment are specific to `ActionPicker` and not something we should abstract over on `ResultList`, which should probably make minimal assumptions on how it's used.

Reply:
Done